en/docs/references/performance-analysis.md

```diff
@@ -1,18 +1,77 @@
+
+#### Latency estimates for individual API invocations in the performance-critical path
+
+The performance analyzer tool displays the estimated latency for each API invocation in the performance-critical path as a banner next to the API invocation in the low-code diagram, as shown below. The Performance Analyzer tool derives these estimates based on the user count range that you have selected.
+
+![Latency estimates for API Invocations in the performce-critical path](../assets/img/references/performance-analysis/latency-estimates-api-invocation.png){.cInlineImage-threeQuarter}
+
+### Analyze the performance of all execution paths
+
+By default, Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click on  **Reveal performance-critical path** on the top banner to highlight it in green. This opens a panel on the right side of the editor. In this panel, you can perform an in-depth performance analysis of the component. 
+
+![In-depth performance analysis](../assets/img/references/performance-analysis/in-depth-analysis.png){.cInlineImage-full}
+
+#### Execution paths
+
+You can view all execution paths in the component in a tabular format listed on top of the **Performance Forecast** panel. The table shows the entries in ascending order of the time it takes to process a single request in each path.
+
+![Execution paths](../assets/img/references/performance-analysis/execution-paths.png){.cInlineImage-half}
+
+The Performance Analyzer tool identifies the execution paths using the service invocations performed by each of them. If an execution path does not include any service invocation, then the latency of that path is assumed to be negligible.
+
+You can select any execution path from the table. When you select an execution path, the Performance Analyzer tool will highlight the execution path on the low-code diagram. The estimated overall performance values displayed on the top banner will change based on the execution path selected.
+
+#### Performance forecast graphs
+
+The performance forecast graphs, throughput, and latency, for the selected execution path, are found below the execution path table in the **Performance Forecast** panel.
+
+![Performance graphs](../assets/img/references/performance-analysis/performance-graphs.png){.cInlineImage-half}
 
-Traditionally, the above is achieved via performance debugging which involves measuring the performance of services by load testing them. The Performance Analyzer allows you to carry out performance measurement at the application development stage, significantly reducing the time and cost spent on performance debugging.
+The throughput and latency graphs are based on the number of users accessing the component simultaneously. By analyzing these graphs carefully, you can gain a fair understanding of how well your component will scale with the increasing load.
 
 ## Assumptions
```

Comment:
This section is relevant to the graphs and is important to know when you are actually reading the graphs. I think it's best to leave it in that section isn't it?

Reply:
But wouldn't the user have already read the graphs by the time they see this? Because they have already gone through the "Use the Performance Analyzer" section?

Reply:
No... they see the graph only when they go to see the critical path.